Tasks/VsTest/VSTest.ps1

```diff
@@ -52,11 +52,18 @@ if($testAssemblyFiles)
 
     $resultFiles = Find-Files -SearchPattern "*.trx" -RootFolder $testResultsDirectory 
 
-    Publish-TestResults -Context $distributedTaskContext -TestResultsFiles $resultFiles -TestRunner "VSTest" -Platform $platform -Configuration $configuration
+    if($resultFiles)
+    {
+        Publish-TestResults -Context $distributedTaskContext -TestResultsFiles $resultFiles -TestRunner "VSTest" -Platform $platform -Configuration $configuration
 
+    }
+    else
+    {
+        Write-Warning "No results found to publish."   
```

Comment:
Shouldn't we have a Boolean property that allows the user to turn off publishing of test results? If so, then the warning makes sense. If not, I am not sure the warning makes sense.